ui/app/AppLayouts/Communities/popups/ExportControlNodePopup.qml

```diff
@@ -4,147 +4,288 @@ import QtQuick.Layouts 1.15
 import QtQml.Models 2.14
 
 import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
 import StatusQ.Core 0.1
 import StatusQ.Popups.Dialog 0.1
 import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
 
 import utils 1.0
 
+import SortFilterProxyModel 0.2
+
 StatusDialog {
     id: root
 
-    property string communityName: ""
-    property string privateKey: ""
-
-    signal deletePrivateKey
+    required property var community
+    property var devicesStore
 
     width: 640
-    title: qsTr("Move %1 community control node").arg(root.communityName)
-
-    closePolicy: Popup.NoAutoClose
 
-    component Paragraph: StatusBaseText {
-        Layout.fillWidth: true
-        Layout.minimumHeight: 40
-        font.pixelSize: Style.current.primaryTextFontSize
-        lineHeightMode: Text.FixedHeight
-        lineHeight: 22
-        wrapMode: Text.Wrap
-        verticalAlignment: Text.AlignVCenter
+    onAboutToShow: {
+        devicesStore.loadDevices()
     }
 
-    component CopyButton: StatusButton {
-        id: copyButton
-        borderColor: textColor
-        disabledTextColor: textColor
-        disabledColor: normalColor
-        text: qsTr("Copy")
-        size: StatusButton.Size.Tiny
-        states: [
-            State {
-                name: "success"
-                PropertyChanges {
-                    target: copyButton
-                    text: ""
-                    icon.name: "checkmark"
-                    normalColor: Theme.palette.successColor2
-                    textColor: Theme.palette.successColor1
-                    enabled: false
+    QtObject {
+        id: d
+        readonly property var devices: SortFilterProxyModel {
+            sourceModel: root.devicesStore.devicesModel
+            sorters: [
+                RoleSorter {
+                    roleName: "isCurrentDevice"
+                    sortOrder: Qt.DescendingOrder
+                    priority: 2
+                },
+                RoleSorter {
+                    roleName: "isMobile"
+                    priority: 1 // Higher number === higher priority
                 }
+            ]
+            proxyRoles: ExpressionRole {
+                name: "isMobile"
+                expression: model.deviceType === "ios" || model.deviceType === "android"
```

Comment:
Maybe you can add an enum or constants to check devices type. I see they are used in here and also in line 52.

Reply:
Yeah... c&p from the Devices settings panel, I can fix it there as well; wdyt @igor-sirotin ?